src/lib.rs

```diff
@@ -19,14 +19,6 @@ pub use c2pa::SigningAlg;
 /// these all need to be public so that the uniffi macro can see them
 mod error;
 pub use error::{Error, Result};
-#[cfg(feature = "v1")]
-mod json_api;
-#[cfg(feature = "v1")]
-pub use json_api::{read_file, read_ingredient_file, sign_file};
-#[cfg(feature = "v1")]
-mod signer_info;
-#[cfg(feature = "v1")]
-pub use signer_info::{CallbackSigner, SignerCallback, SignerConfig, SignerInfo};
```

Comment:
This features had already been removed from the crate. Removing the code so clippy is happy :)